spec/integration/query_cache_spec.rb

```diff
@@ -513,4 +513,155 @@
       expect(events.length).to eq(2)
     end
   end
+
+  describe 'in transactions' do
+    require_transaction_support
+
+    let(:collection) { authorized_client['test'] }
+
+    let(:events) do
+      subscriber.command_started_events('find')
+    end
+
+    before do
+      collection.delete_many
```

Comment:
What do you think about making a helper like `create_collection` that would drop & create? Should take care of red CI then I expect.